Recipe-App/Recipe-App/UIViews/RecipeListView.swift

```diff
@@ -44,9 +44,11 @@ struct RecipeListView: View {
                     //This is where the list data will go
                     ScrollView(.vertical, showsIndicators: false) {
                         ForEach(placeholderData, id: \.self) { text in
```

Comment:
Your loop can look like this
                        ForEach(AppConstants.dummyRecipeList, id: \.self) { recipe in
                            RecipeCardItemView(recipe: recipe)
                    }